app/src/main/java/de/tum/in/tumcampusapp/activities/PersonsDetailsActivity.java

```diff
@@ -418,15 +418,16 @@ private boolean isPermissionGranted(int id) {
 
                 // Display an AlertDialog with an explanation and a button to trigger the request.
                 new AlertDialog.Builder(this)
-                        //.setMessage(getString(R.string.permission_calendar_explanation))
-                        .setMessage("Lorem Ipsum")
```

Comment:
We still had a Lorem Ipsum in there? And git blame even tells me it was my commit... 🤦‍
Good catch

Reply:
tztztztzzt - better add a todo marker next time - just sayin'